src/subcommands/publishers.rs

```diff
@@ -3,32 +3,47 @@ use std::collections::BTreeMap;
 use crate::publishers::fetch_owners_of_crates;
 use crate::{common::*, publishers::PublisherData};
 
-pub fn publishers(args: Vec<String>, max_age: std::time::Duration) -> Result<(), std::io::Error> {
-    let dependencies = sourced_dependencies(args);
+pub fn publishers(
+    metadata_args: Vec<String>,
+    diffable: bool,
+    max_age: std::time::Duration,
+) -> Result<(), std::io::Error> {
+    let dependencies = sourced_dependencies(metadata_args);
     complain_about_non_crates_io_crates(&dependencies);
     let (publisher_users, publisher_teams) = fetch_owners_of_crates(&dependencies, max_age)?;
 
-    if !publisher_users.is_empty() {
+    // Group data by user rather than by crate
+    let mut user_to_crate_map = transpose_publishers_map(&publisher_users);
+    let mut team_to_crate_map = transpose_publishers_map(&publisher_teams);
+
+    // Sort crate names alphabetically
+    user_to_crate_map.values_mut().for_each(|c| c.sort());
+    team_to_crate_map.values_mut().for_each(|c| c.sort());
+
+    if !publisher_users.is_empty() && !diffable {
         println!("\nThe following individuals can publish updates for your dependencies:\n");
-        let user_to_crate_map = transpose_publishers_map(&publisher_users);
         let map_for_display = sort_transposed_map_for_display(user_to_crate_map);
         for (i, (user, crates)) in map_for_display.iter().enumerate() {
             // We do not print usernames, since you can embed terminal control sequences in them
             // and erase yourself from the output that way.
-            // TODO: check if it's possible to smuggle those into github/crates.io usernames
             let crate_list = comma_separated_list(&crates);
             println!(" {}. {} via crates: {}", i + 1, &user.login, crate_list);
         }
+        println!("\nNote: there may be outstanding publisher invitations. crates.io provides no way to list them.");
+        println!("See https://github.com/rust-lang/crates.io/issues/2868 for more info.");
```

Comment:
I'd be more consistent with other warnings in `common.rs` if this were an `eprintln!` as we expect to remove it in the future.